js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,13 @@
+function aperture(n, arr) {
+  let arr_length = arr.length;
+  if (n > arr_length) return [];
```

Comment:
you don't need this condition. for loop won't execute if n > array length and it will automatically return an empty array.

Reply:
thanks for feedback